_data/simple-icons.json

```diff
@@ -929,6 +929,11 @@
             "hex": "3499CD",
             "source": "https://automattic.com/press/brand-materials/"
         },
+        {
+            "title": "Autoprefixer",
+            "hex": "DD3735",
+            "source": "https://github.com/postcss/autoprefixer"
```

Comment:
Can you change the source for https://github.com/postcss/autoprefixer/blob/1341747bc8142a147342f55eea5ed4286a3ca318/logo.svg